client_local_api/bootstrap.go

```diff
@@ -0,0 +1,38 @@
+package client_local_api
+
+import (
+	log "github.com/cihub/seelog"
+	"github.com/mysterium/node/client_local_api/endpoints"
+	"net/http"
+	"reflect"
+	"runtime"
+)
+
+const httpLogPrefix = "[http]"
+
+func registerAllEndpoints() {
+	RegisterEndpoint("/healthcheck", endpoints.HealthCheckEndpoint)
+}
+
+/*
+Bootstrap function starts http server on specified binding address, which format conforms to what is expeted by
+http.ListenAndServe function. This function IS BLOCKING, that means - it should be run on goroutine to resume with normal flow
+*/
+func Bootstrap(bindAddress string) {
```

Comment:
Are You sure it's enough to have global functions in package.
Having wrapper like would make testing easier:
```
TekilaServer {
    host string
    port int
    routes []Handler
} 
```